docs/pages/database-access/enroll-aws-databases/redshift-serverless.mdx

```diff
@@ -0,0 +1,363 @@
+---
+title: Database Access with Redshift Serverless on AWS
+description: How to configure Teleport database access with Amazon Redshift Serverless.
+---
+
+(!docs/pages/includes/database-access/db-introduction.mdx  dbType="Amazon Redshift Serverless" dbConfigure="Amazon Redshift Serverless database with IAM authentication" dbName="Amazon Redshift Serverless" !)
+
+This guide will help you to:
+
+- Set up Teleport to access your Amazon Redshift Serverless workgroups.
+- Connect to your databases through Teleport.
+
+<Tabs>
+<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+![Teleport Database Access Redshift Self-Hosted](../../../img/database-access/guides/redshift_selfhosted_serverless.png)
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+![Teleport Database Access Redshift Cloud](../../../img/database-access/guides/redshift_cloud_serverless.png)
+</TabItem>
+
+</Tabs>
+
+This guide shows how to enroll a single Amazon Redshift Serverless database
+through the Teleport Database Service. For a more scalable approach, learn how
+to [configure the Teleport Discovery
+Service](../../auto-discovery/databases.mdx) to automatically enroll all AWS
+databases in your infrastructure.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- AWS account with a Redshift Serverless configuration and permissions to create
+  and attach IAM policies.
+- Command-line client `psql` installed and added to your system's `PATH` environment variable.
+- A host where you will run the Teleport Database Service. This guide assumes an
+  EC2 instance, and provides a corresponding example of access control.
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/5. Create an IAM policy for Teleport
```

Comment:
IIRC, this section was written by Alex to list out all the IAM permissions that are required for running the bootstrap command. 

I would suggest removing this step to be more aligned with other guides where we don't show this detail, assuming most users using the bootstrap command are admins. 

I can add this to the AWS IAM reference page when I work on it. WDYT?

Reply:
It looks like the AWS database guides are pretty inconsistent here. Some guides include the `docs/pages/includes/database-access/aws-bootstrap.mdx` partial, which includes both manual and automatic options for the `teleport db configure bootstrap` command, but don't include an IAM policy to enable automatic IAM bootstrapping.

Would it make sense to add the IAM policy for automatic bootstrapping to the `aws-bootstrap.mdx` partial? If it makes more sense, we could also add the policy to another partial and include that partial in all of the pages that use the `aws-bootstrap.mdx` partial. We could then edit this page to follow the same structure.

How does that sound?

Reply:
> Would it make sense to add the IAM policy for automatic bootstrapping to the aws-bootstrap.mdx partial? If it makes more sense, we could also add the policy to another partial and include that partial in all of the pages that use the aws-bootstrap.mdx partial. We could then edit this page to follow the same structure.
> How does that sound?

Sure. It would be beneficial for those who want to know the exact IAM permissions that are required for bootstrapping. But I wouldn't create a new IAM role/user with those permissions just to run the bootstrap command. Normally you should just run bootstrapping as an admin. 

> It looks like the AWS database guides are pretty inconsistent here.

Agreed. There is also a whole separate discussion about whether we should suggest bootstrapping at all since it's not very easy to use. Bootstrapping may be ok for a simple static URL like the new guide. But bootstrapping is very complicated for auto-discovery and cross-account support. I was planning to address that separately, either always using `teleport db configure aws print-iam`  or just listing out the exact permissions in IAM JSON format.

Reply:
That makes sense! Just to make sure I understand, the recommended move across the AWS database access guides is to:
- Only show manual bootstrap command instructions
- Require that the user have admin permissions on AWS
- Remove IAM policies for bootstrapping

Reply:
> Remove IAM policies for bootstrapping
Yes

> Only show manual bootstrap command instructions
> Require that the user have admin permissions on AWS

These two are either-or. If we require the reader to have admin permissions, then they can just use "automatic" bootstrap. 

For the next round, I guess i will list out the actual IAM permissions required for each database type in each guide similar to this https://goteleport.com/docs/auto-discovery/servers/ec2-discovery/#step-27-define-an-iam-policy, and have a separate guide/reference on bootstrapping

Reply:
Sounds good! 👍